portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,73 @@
 <!DOCTYPE html>
 <html>
+    
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <!-- <style>
```

Comment:
In general, we only check in comments that provide us value - commented out code is pretty consistently not that, so I'd remove this style tag.

Reply:
Same throughout this document.